src/main/java/com/zendesk/maxwell/replication/Position.java

```diff
@@ -69,8 +67,10 @@ public int hashCode() {
 	}
 
 	public boolean newerThan(Position other) {
-		if ( other == null )
-			return true;
-		return this.getBinlogPosition().newerThan(other.getBinlogPosition());
+		if (other instanceof Position) {
+			Position mOther = (Position) other;
+			return binlogPosition.newerThan(mOther.binlogPosition);
+		}
+		return true;
```

Comment:
I think this function read better before...

Reply:
Sorry, those are artifacts from my attempt to refactor position classes in a slightly different manner, they don't make much sense anymore.  Thank you for spotting those, I've reverted the changes now.